webextensions/api/tabs.json

```diff
@@ -2855,7 +2852,7 @@
               "__compat": {
                 "support": {
                   "chrome": {
-                    "version_added": "≤61"
+                    "version_added": "54"
```

Comment:
✅ https://github.com/chromium/chromium/commit/94afe44e219ad774df9290dd980d6214141a089a in [54.0.2813.0](https://github.com/chromium/chromium/releases/tag/54.0.2813.0) before 54.0.2840.